### PR TITLE
Bug with aggregation

### PR DESCRIPTION
**Pull request by rahularya50**

### CODE REVIEW

File: static/js/interpreter.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ export function initializeInterpreter(divName) {
         editor.commands.addCommand({
             name: "ctrl-enter",
             bindKey: { win: "Ctrl+Enter", mac: "Cmd+Enter"},
-            exec: function(editor, ...rest) {run(editor.getValue().replace(/\r/g, ""), db);}
```
Comment (kavigupta) [inline static/js/interpreter.js]: What does this have to do with aggregation

**Review by kavigupta — CHANGES_REQUESTED** [review]

File: static/js/interpreter.js (hunk 1 above)
Comment (rahularya50) [reply static/js/interpreter.js]: uhh it's also a bug that I noticed while testing an aggregation?

**Review by kavigupta — APPROVED** [review]
no idea what this is but should be good
